src/main/java/Mars.java

```diff
@@ -0,0 +1,205 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
+
+public class Mars {
+    public static void main(String[] args) {
+        List<Task> arrayList = new ArrayList<>();
+        /* PRE- COMMITS
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+         */
+
+        /* LEVEL 0. Rename, Greet, Exit
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Mars\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n" +
+                " Bye. Hope to see you again soon!\n" +
+                "____________________________________________________________"); */
+        try{
+            Mars mars = new Mars();
+            // mars.echo();
+            //mars.store(arrayList);
+            // mars.list(arrayList);
+            // mars.mark(arrayList);
+            // mars.unmark(arrayList);
+            mars.addTask(arrayList);
+            mars.deleteTask(arrayList);
+        } catch (marsException e) {
+            System.out.println(e.getMessage());
+        }
+        finally {
+            ;
+        }
+
+
+
+    }
+
+    /*Level 1. Echo*/
+    private void echo(){
+        System.out.println(" Hello! I'm Mars\n What can I do for you?\n");
+        Scanner reader = new Scanner(System.in);
+        String n = reader.nextLine();
+
+        while(!n.equals("bye")){
+            System.out.println("____________________________________________________________\n" +
+                                n +
+                                "\n____________________________________________________________\n");
+            n = reader.nextLine();
+        }
+        System.out.println("Bye. Hope to see you again soon!");
+        // reader.close();
+    }
+
+    /*Level 2. Add, List */
+    private void store(List<Task> arrayList){
+        System.out.println(" Hello! I'm Mars\n What can I do for you?\n");
+        Scanner reader = new Scanner(System.in);
+        String n = reader.nextLine();
+        while(!n.equals("bye")){
+            if(n.equals("list")){
+                Mars.list(arrayList);
+            }
+            else {
```

Comment:
```suggestion
            } else {
```

Should be modified to follow the coding conventions.